src/snapshots/expr/tuple_unification_test.md

```diff
@@ -0,0 +1,62 @@
+# META
+~~~ini
+description=Tuple unification with different types
+type=expr
+~~~
+# SOURCE
+~~~roc
+[(1, "a"), (2.5, "b")]
+~~~
+# PROBLEMS
+**TYPE MISMATCH**
```

Comment:
This snapshot is reproducing a bug that's unrelated to this change. I'll fix it in a future PR!

Reply:
nm fixed it in 2925739fea0cf91adaadb5c215281798ac441743